Landing/script.js

```diff
@@ -0,0 +1,34 @@
+function navLinkActive(event) {
+  const a = document.querySelector(".nav-link--active");
+  event.stopPropagation();
+  event.target.classList.add("nav-link--active");
+  a.classList.remove("nav-link--active");
+  console.log(event);
+}
+
+const links = document.querySelector(".nav-box");
+
+links.addEventListener("click", navLinkActive, true);
+
+let btns = document.querySelectorAll("*[data-modal-btn]");
+
+for (let i = 0; i < btns.length; i++) {
+  btns[i].addEventListener("click", function () {
+    let name = btns[i].getAttribute("data-modal-btn");
```

Comment:
1. This variable is not changes. Use const
2. If you decided to use data attribute you can get it's value with `dataset` property